async_simple/executors/test/SimpleIOExecutorTest.cpp

```diff
@@ -51,21 +51,21 @@ class SimpleIOExecutorTest : public FUTURE_TESTBASE {
 
 TEST_F(SimpleIOExecutorTest, testNormal) {
     std::string expect(4096, '0');
-    auto fd = open(kTestFile, O_RDWR | O_DIRECT | O_CREAT, 0600);
+    auto fd = open(kTestFile, O_RDWR | O_DIRECT | O_CREAT);
     auto output = memalign(4096, kBufferSize);
     memcpy((char*)output, expect.data(), expect.length());
     _executor->submitIO(
         fd, IOCB_CMD_PWRITE, output, expect.length(), 0,
-        [](io_event& event) mutable { EXPECT_EQ(4096, (int32_t)event.res); });
-    std::this_thread::sleep_for(300ms);
+        [](io_event_t& event) mutable { EXPECT_EQ(4096, (int32_t)event.res); });
+    usleep(1000 * 300);
```

Comment:
This change is not intentional, right?

Reply:
Maybe it's from an old version, i will confirm.